examples/gottbert/README.md

```diff
@@ -0,0 +1,64 @@
+# GottBERT: a pure German language model
+
+## Introduction
+
+[GottBERT](http://arxiv.org/abs/2012.02110) is a pretrained language model trained on 145GB of German text based on RoBERTa.
+
+## Example usage
+
+### fairseq
+##### Load GottBERT from torch.hub (PyTorch >= 1.1):
+```python
+import torch
+gottbert = torch.hub.load('pytorch/fairseq', 'gottbert')
```

Comment:
I think this should be `gottbert-base` right?

Reply:
Yep, indeed. Thank you! :)

Reply:
The extra line in the roberta readme should have already been added since the initial merge request. However, I changed the style of the entry to conform with the others (added : ).